code/modules/mob/living/simple_animal/hostile/thing.dm

```diff
@@ -0,0 +1,2046 @@
+	else
+		return TRUE
+
+/mob/living/simple_animal/hostile/thing/update_sight()
+	..()
+	if(can_omnivision)
+		lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE // Can still see some darkness
+		sight = (SEE_MOBS|SEE_OBJS|SEE_TURFS)
+		see_in_dark = 8
+	else
+		lighting_alpha = initial(lighting_alpha)
+		see_in_dark = initial(see_in_dark)
+		sight = initial(sight)
+
+/mob/living/simple_animal/hostile/thing/proc/Evolve() // No verb for this one
+	if(stat == DEAD)
+		return
+	if(phasing)
+		return
+	if(absorbing)
+		return
+	if(prying)
+		return
+	if(can_evolve == 0)
+		return
+	if(absorptions <= 44)
+		return
+	canmove = 0
+	visible_message("<span class='danger'>\the [src] suddenly stands still, twitching ever so slightly.</span>")
+	absorbing = 1 // Easy way to stop it from using any abilities
+	sleep(100)
+	var/mob/living/simple_animal/hostile/bigthing/B = new /mob/living/simple_animal/hostile/bigthing(loc)
+	if(mind)
+		mind.transfer_to(B)
+	B.dir = dir
+	visible_message("<span class='biggerdanger'>\the [src] creaks and convulses, as it grows in size and tentacles emerge from its back!</span>")
+	playsound(loc, 'sound/thing/attack.ogg', 80, 1)
+	qdel(src)
+
+/mob/living/simple_animal/hostile/thing/emote(act, m_type = 1, message = null, force)
+	if(stat == DEAD)
+		return
+
+	act = lowertext(act)
+
+	switch(act)
+
+		if("roar")
+			if(sound_cooldown <= 0)
+				if(!phasing) // Won't show what did the sound if the thing is invisible
+					message = "<B>\The [src]</B> snarls... and then roars!"
+					m_type = 2
+				else
+					message = "<B>A strange snarling noise fills the room... and suddenly a loud noise!</b>"
+					m_type = 2
+		if("growl")
+			if(sound_cooldown <= 0)
+				if(!phasing) // Won't show what did the sound if the thing is invisible
+					message = "<B>\The [src]</B> growls..."
+					m_type = 2
+				else
+					message = "<B>A strange growling noise fills the room...</b>"
+					m_type = 2
+		if("help")
+			to_chat(src, "growl, roar")
+
+	if((!stat) && (sound_cooldown <= 0))
+		if(act == "roar")
+			playsound(src, 'sound/thing/roar.ogg', 70, 1, 500)
+			sound_cooldown = initial(sound_cooldown)
+		if(act == "growl")
+			playsound(loc, "growl", 100, 1)
+			sound_cooldown = initial(sound_cooldown)
+		..()
+
+/mob/living/simple_animal/hostile/thing/verb/Empower() // Verb to buy abilities
+	set name = "Empower"
+	set category = "Unknown"
+	set desc = "Unlock more abilities."
+
+	Empowerproc()
+
+#define THING_TRAVEL "Travel - Activable - 5 points"
+#define THING_WEAPONIZE "Weaponize - Activable - 5 points"
+#define THING_CONVERT "Convert - Passive - 10 points"
+#define THING_OMNIVISION "Omnivision - Passive - 5 points"
+#define THING_SCREECH "Screech -  Activable - 5 points"
+#define THING_SCREECH_UPGRADE "Screech Upgrade - 10 points"
+#define THING_EMP "Electromagnetic Disturbance - Activable - 5 points"
+#define THING_EMP_UPGRADE "Electromagnetic Disturbance Upgrade - 10 points"
+#define THING_BUILD "Build - Activable - 10 points"
```

Comment:
Define should be reserved for macro that make things more easily readable. Or avoid magical number in the code.

This is not one of them. This make the code LESS readable because the reader has to go back up to figure out what your plain text string are.